receiver/sqlserverreceiver/metadata.yaml

```
@@ -43,6 +43,9 @@ attributes:
     description: The page operation types.
     type: string
     enum: [read, write]
+  page.performance_objects:
```

```suggestion
  performance_counter.object_name:
```
It may make sense to generalize this more, as it may end up being useful for other metrics if the same error is hit. This isn't specific to pages, but a general attribute for all performance counters.